crypto/mem_clr.c

```diff
@@ -60,6 +60,7 @@
 #include <string.h>
 #include <openssl/crypto.h>
 
+extern unsigned char cleanse_ctr;
```

Comment:
Indeed, the warning was spotted in the gcc-5 and clang-3.6 sanitized builds:
https://travis-ci.org/openssl/openssl/jobs/87041410
https://travis-ci.org/openssl/openssl/jobs/87041412